Rancher PAYG offerings for AWS and Azure (#1008)
rancher/rancher-docs: docs/integrations-in-rancher/cloud-marketplace/aws-marketplace-payg-integration/upgrading-rancher-payg-cluster.md

---
title: Upgrading Rancher PAYG Cluster
---

> ## Upgrading a Rancher Prime PAYG Cluster

Reviewer: ```suggestion
```

> The marketplace PAYG offer is tied to a billing adapter AND Rancher Prime version. These are updated periodically as new version of the billing adapter or Rancher are released.
> In that case the helm chart will be updated with new tags and digests and new version of helm chart will be uploaded. In order to upgrade the deployed helm chart with new version, execute the following helm command

Reviewer: ```suggestion
The Amazon marketplace pay-as-you-go (PAYG) offering is tied to both a billing adapter and a particular Rancher Prime version. These are updated periodically, whenever there is a new version available of the billing adapter or of Rancher. When there is a new update, the Helm chart will also be updated with new tags and digests, and a new version of the Helm chart will be uploaded. To upgrade the deployed Helm chart to the latest version, run the following:
```

> ```

Reviewer: ```suggestion
```shell
```

helm upgrade -n cattle-rancher-csp-deployer-system rancher-cloud --create-namespace \

> oci://709825985650.dkr.ecr.us-east-1.amazonaws.com/suse/{{repository}}/rancher-cloud-helm/rancher-cloud \

Reviewer 2: ```suggestion
oci://709825985650.dkr.ecr.us-east-1.amazonaws.com/suse/$REPOSITORY/rancher-cloud-helm/rancher-cloud \
```

>   --version <upgraded_chart_version> \

Reviewer 2: ```suggestion
  --version $UPGRADED_CHART_VERSION \
```

  --set rancherHostname=$HOST_NAME \
  --set rancherServerURL=https://$HOST_NAME \
  --set rancherReplicas=$REPLICAS \
  --set rancherIngressClassName=nginx \
  --set global.aws.accountNumber=$AWS_ACCOUNT_ID \
  --set global.aws.roleName=$ROLE_NAME
```

> To check if upgraded helm chart installation is deployed, run following command. It should display the upgraded chart version and REVISION incremented by 1 from previous install.

Reviewer: ```suggestion
To check if the upgraded Helm chart deployed successfully:
```

> ```

Reviewer: ```suggestion
```shell
```

helm ls -n cattle-rancher-csp-deployer-system

> ```

Reviewer: Suggest moving "This command should display the upgraded chart version and revision, incremented by one from previous install." to below the command that ends on line 26.

---
title: Upgrading Rancher Prime PAYG Cluster in AWS
---

The AWS Marketplace PAYG offering is tied to a billing adapter and the Rancher Prime version. These are updated periodically as new version of the billing adapter or Rancher Prime are released. In this situation, the helm chart will be updated with new tags and digests, and a new version of the helm chart will be uploaded.

To upgrade the deployed helm chart with the latest version, run the following helm command:

```shell
helm upgrade -n cattle-rancher-csp-deployer-system rancher-cloud --create-namespace \
oci://709825985650.dkr.ecr.us-east-1.amazonaws.com/suse/$REPOSITORY/rancher-cloud-helm/rancher-cloud \
  --version $UPGRADED_CHART_VERSION \
  --set rancherHostname=$HOST_NAME \
  --set rancherServerURL=https://$HOST_NAME \
  --set rancherReplicas=$REPLICAS \
  --set rancherIngressClassName=nginx \
  --set global.aws.accountNumber=$AWS_ACCOUNT_ID \
  --set global.aws.roleName=$ROLE_NAME
```

To check if the upgraded Helm chart deployed successfully:

```shell
helm ls -n cattle-rancher-csp-deployer-system
```

> :::note

Reviewer 2: ```suggestion
:::warning
```
